core/src/main/java/com/linecorp/armeria/server/annotation/Attribute.java

```diff
@@ -0,0 +1,16 @@
+package com.linecorp.armeria.server.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.linecorp.armeria.internal.server.annotation.DefaultValues;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.PARAMETER)
+public @interface Attribute {
+
+    String value();
+    Class<?> prefix() default DefaultValues.class;
+}
```

Comment:
Please add Javadoc. Check other annotations such as `@Param` and `@Header` for examples.

Reply:
I update it!